docs/OpenShift GitOps Usage Guide.md

```diff
@@ -412,6 +415,50 @@ Make sure to click **Save**. You should now have a new tab called **Credentials*
 
 ![image alt text](assets/16.credentials_setup.png)
 
+## **Setting environment variables**
+
+Updating the following environment variables in the existing Subscription Object for the GitOps Operator will allow you (as an admin) to change certain properties in your cluster:
+
+<table>
+  <tr>
+    <td>Environment variable</td>
+    <td>Default value</td>
+    <td>Description</td>
+  </tr>
+  <tr>
+    <td>ARGOCD_CLUSTER_CONFIG_NAMESPACES</td>
+    <td>none</td>
+    <td>When provided with a namespace, Argo CD is granted permissions to manage specific cluster-scoped resources which include
+    platform operators, optional OLM operators, user management, etc. Argo CD is not granted cluster-admin.</td>
+  </tr>
+  <tr>
+    <td>CONTROLLER_CLUSTER_ROLE</td>
+    <td>none</td>
+    <td>Administrators can configure a common cluster role for all the managed namespaces in role bindings for the Argo CD application controller with this environment variable. Note: If this environment variable contains custom roles, the Operator doesn't create the default admin role. Instead, it uses the existing custom role for all managed namespaces.</td>
```

Comment:
should this be `custom` instead of `common`?

Reply:
I got this from line 864 `With this update, administrators can configure a common cluster role for all the managed namespaces.`, do you think it should be common or cluster?

Reply:
thats good enough for me 
resolving this